lib/rules.js

```diff
@@ -184,6 +227,185 @@ export const MULTIPLY_BY_INVERSE = defineRuleString('#a / (#b / #c)', '#a * (#c
 // e.g. |-3| -> 3
 export const ABSOLUTE_VALUE = defineRuleString('|-#a|', '#a')
 
+// ROOT
+
+// assumes that the exponent is a constant value
+// TODO: handle cases like nthRoot('x^2y', 2)
+// e.g. nthRoot(x^2, 4) -> nthRoot(x^1 ,2)
+export const CANCEL_EXPONENT = defineRule(
+    (node) => {
+        let isNthRoot = false
+        let validRadicand = false
+        if (query.isApply(node)) {
+            isNthRoot = (node.op == 'nthRoot')
+            validRadicand = query.isPow(node.args[0])
+        }
+        return (isNthRoot && validRadicand) ? {node} : null
+    },
+    (node) => {
+        const radicand = node.args[0]
+        const variable = radicand.args[0]
+        const exponent = radicand.args[1]
+        let index = node.args[1]
+
+        // simplify exponent / index
+        // e.g. nthRoot(x^2, 4) -> 2/4 -> 1/2
+        const newRoot = applyRule(SIMPLIFY_FRACTION,
+                                  build.applyNode('div', [exponent, index]))
+
+        let numerator = newRoot.args[0]
+        let denominator = newRoot.args[1]
+        let numVal = query.getValue(numerator)
+        let denomVal = query.getValue(denominator)
```

Comment:
Calling these `numerator` and `denominator` is a bit misleading.  Can you rename these to `exponent` and `index`?